core/src/avm2/globals/object.rs

```diff
@@ -274,15 +274,43 @@ pub fn create_class<'gc>(activation: &mut Activation<'_, 'gc>) -> GcCell<'gc, Cl
     ));
 
     // Fixed traits (in AS3 namespace)
-    const AS3_INSTANCE_METHODS: &[(&str, NativeMethodImpl)] = &[
-        ("hasOwnProperty", has_own_property),
-        ("isPrototypeOf", is_prototype_of),
-        ("propertyIsEnumerable", property_is_enumerable),
+    let as3_instance_methods: Vec<(&str, NativeMethodImpl, _, _)> = vec![
+        // These signatures are weird, but they match the describeTypeJSON output
+        (
+            "hasOwnProperty",
+            has_own_property,
+            vec![ParamConfig::optional(
+                "name",
+                Multiname::any(activation.context.gc_context),
+                true,
```

Comment:
wait, the default value of the param is `true`? not `undefined`?

Reply:
oops - fixed